### PR TITLE
added notification when uncordoning

### PR DESCRIPTION
**Pull request by atighineanu**
when reboot & uncordoning is successful -> notification will be sent.
 Fixes: https://github.com/weaveworks/kured/issues/585
 
AFTER:
```
[kured-bot](https://app.slack.com/team)
APP  
[1:26 PM](https://kured-tesing-personal.slack.com/archives)
Draining node kind-worker2
[1:26](https://kured-tesing-personal.slack.com/archives)
Rebooting node kind-worker2
[1:27](https://kured-tesing-personal.slack.com/archives)
Node: kind-worker2 rebooted&uncordoned successfully!
```

### CODE REVIEW

File: cmd/kured/main.go [hunk 1]
```diff
@@ -635,6 +635,12 @@ func rebootAsRequired(nodeID string, rebootCommand []string, sentinelCommand []s
 				if err != nil {
 					log.Errorf("Unable to uncordon %s: %v, will continue to hold lock and retry uncordon", node.GetName(), err)
 					continue
+				} else {
+					if notifyURL != "" {
+						if err := shoutrrr.Send(notifyURL, fmt.Sprintf("Node: %s rebooted&uncordoned successfully!", nodeID)); err != nil {
```
Comment (jackfrancis) [inline cmd/kured/main.go:640]: nit: add a space so it reads "rebooted & uncordoned successfully!"

Comment (atighineanu) [reply cmd/kured/main.go]: Done :+1:

**Review by jackfrancis — APPROVED** [review]
lgtm

**Comment by ckotzbauer** [comment]
That looks good, thanks. I just would propose to introduce a message-template parameter as we have it already for the other notifications to stay in sync.

**Comment by atighineanu** [comment]
@ckotzbauer  
AFTER:
[kured-bot](https://app.slack.com/team/B01F49PD9EV)
APP  [2:00 PM](https://kured-tesing-personal.slack.com/archives/C01ERSH2PFD/p1656072042556409)
Draining node kind-worker
[2:00](https://kured-tesing-personal.slack.com/archives/C01ERSH2PFD/p1656072042789399)
Rebooting node kind-worker
New
[2:02](https://kured-tesing-personal.slack.com/archives/C01ERSH2PFD/p1656072131504699)
Node kind-worker rebooted & uncordoned successfully!

**Comment by ckotzbauer** [comment]
Great, thanks for the flag.
Can you please add it to the README [here](https://github.com/weaveworks/kured#notifications) and [here](https://github.com/weaveworks/kured#configuration) as well? Then I'm finally happy 😆

**Comment by atighineanu** [comment]
> Great, thanks for the flag. Can you please add it to the README [here](https://github.com/weaveworks/kured#notifications) and [here](https://github.com/weaveworks/kured#configuration) as well? Then I'm finally happy laughing

Done. 
Readme, and charts/values as well.

**Comment by jackfrancis** [comment]
@ckotzbauer thanks for making this PR even better :)

**Comment by ckotzbauer** [comment]
Thanks for the readme change @atighineanu.
Sorry again, but can we do the chart-values change in a separate PR? We have to split chart- and non-chart-changes due to the release-process.

**Comment by atighineanu** [comment]
> Thanks for the readme change @atighineanu. Sorry again, but can we do the chart-values change in a separate PR? We have to split chart- and non-chart-changes due to the release-process.

DONE

**Review by ckotzbauer — APPROVED** [review]

**Pull request merged**